tests/test_cookies.py

```diff
@@ -344,3 +344,46 @@ def test_bake_should_fail(cookies):
     result.stdout.fnmatch_lines([
         '*::test_bake_should_fail PASSED*',
     ])
+
+@pytest.mark.parametrize('choice', ['mkdocs', 'sphinx', 'none'])
+def test_cookies_bake_choices(testdir, choice):
+    """Programmatically create a **Cookiecutter** template and make sure that
+    cookies.bake() works with choice variables.
+    """
+    template = testdir.tmpdir.ensure('cookiecutter-choices', dir=True)
+    template_config = {
+        'repo_name': 'docs',
+        'docs_tool': ['mkdocs', 'sphinx', 'none'],
+    }
+    template.join('cookiecutter.json').write(json.dumps(template_config))
+
+    repo = template.ensure('{{cookiecutter.repo_name}}', dir=True)
+    repo.join('README.rst').write("docs_tool: {{cookiecutter.docs_tool}}")
+
+    testdir.makepyfile("""
+        # -*- coding: utf-8 -*-
+
+        def test_bake_project(cookies):
+            result = cookies.bake(
+                extra_context={'docs_tool': '%s'},
```

Comment:
In #24, the example has just a string for the choice, not a list?

Reply:
The pytest marker injects a string into the ``choice`` variable.